code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -1,285 +1,646 @@
+	slowdown = 0
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS // The martial art is literally based around fire JFC do you want to burn?
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/toggle_spacesuit, /datum/action/item_action/initialize_clock_suit)
+
+	//Using ninja's suit code to make the Cuirass lock onto you
+	var/mob/living/carbon/human/affecting = null
+	var/s_initialized = 0
+	var/s_busy = FALSE
+	var/s_delay = 40
+	var/obj/item/clothing/shoes/combat/swat/bronze/c_shoes
+	var/obj/item/clothing/head/helmet/space/clockwork/c_helmet
+	var/obj/item/clothing/gloves/combat/clockwork/c_gauntlets
+	var/obj/item/clothing/glasses/hud/clock/c_visor
+
+/datum/action/item_action/initialize_clock_suit
+	name = "Activate Cuirass"
+
+#define CLOCK_SUIT_TRAIT "clockwork-cuirass"
+
+// Initiailizing it gives you NODROP
+
+/obj/item/clothing/suit/space/clockwork/proc/lock_suit(mob/living/carbon/human/H)
+	if(!istype(H))
+		return FALSE
+	if(!istype(H.head, /obj/item/clothing/head/helmet/space/clockwork))
+		to_chat(H, "<span class='userdanger'/span> The Justiciar requires you to have your helmet to unlock your potential.")
+		return FALSE
+	if(!istype(H.shoes, /obj/item/clothing/shoes/combat/swat/bronze))
+		to_chat(H, "<span class='userdanger'/span> The clockwork Justiciar requires you to wear his treads.")
+		return FALSE
+	if(!istype(H.gloves, /obj/item/clothing/gloves/combat/clockwork))
+		to_chat(H, "<span class='userdanger'/span> Brass gauntlets are essential to enable your power.")
+		return FALSE
+	if(!istype(H.glasses, /obj/item/clothing/glasses/hud/clock))
+		to_chat(H, "<span class='userdanger'/span> Unlock your True Sight through my visor before activating this.")
+		return FALSE
+	affecting = H
+	ADD_TRAIT(src, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	c_helmet = H.head
+	ADD_TRAIT(c_helmet, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	c_shoes = H.shoes
+	ADD_TRAIT(c_shoes, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	c_gauntlets = H.gloves
+	ADD_TRAIT(c_gauntlets, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	c_visor = H.glasses
+	ADD_TRAIT(c_visor, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	return TRUE
+
+/obj/item/clothing/suit/space/clockwork/proc/unlock_suit()
+	affecting = null
+	REMOVE_TRAIT(src, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	if(c_helmet)//Should be attached, might not be attached.
+		REMOVE_TRAIT(c_helmet, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	if(c_shoes)
+		REMOVE_TRAIT(c_shoes, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	if(c_gauntlets)
+		REMOVE_TRAIT(c_gauntlets, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+
+// Initialize messages
+
+#define CLOCK_LOCK_PHASE 				1
+#define CLOCK_COMPLETE_PHASE	 		6
+#define CLOCK_DEINIT_LOGOFF_PHASE		1
+
+GLOBAL_LIST_INIT(clock_initialize_messages, list(
+	"Redirecting the remains of the Justiciar...",
+	"Securing hardsuit...\nConnection established.",
+	"Finding Reebee...",
```

Comment:
correct me if I'm wrong (I wasn't around back when clock cult was in rotation), but wasn't it Reebe, not Reebee?

Reply:
also, might wanna replace "finding" with "locating", it sounds cooler

Reply:
fUck you're right aaaaa I have to fix a lot of stuff apart from what I'm already adding on my commit